src/Chapter15/Listing15.22.CreatingAChildCollectionWithSystem.Linq.Enumerable.GroupJoin.cs

```diff
@@ -23,14 +28,18 @@ public static void Main()
                         departmentEmployees
                     ));
 
-            foreach ((_, string name, IEnumerable<Employee> employeeCollection) in items)
+            foreach (
+                (_, string name, IEnumerable<Employee> employeeCollection) 
+                    in items)
```

Comment:
I think this can be made into one line

Reply:
Too long otherwise

Reply:
I changed it to be how it is written in the book, it's not a single line there